types/stripe-js/elements.d.ts

```diff
@@ -76,6 +77,23 @@ declare module '@stripe/stripe-js' {
      */
     getElement(elementType: 'cardCvc'): StripeCardCvcElement | null;
 
+    /////////////////////////////
+    /// fpxBank
+    /////////////////////////////
+
+    /**
+     * Creates an `FpxBankElement`.
+     */
+    create(
+      elementType: 'fpxBank',
+      options?: StripeFpxBankElementOptions
```

Comment:
```suggestion
      options: StripeFpxBankElementOptions
```
I think `options` must be passed with at least a `accountHolderType` field.